hack/dind

```diff
@@ -25,6 +25,12 @@ if ! mountpoint -q /tmp; then
 	mount -t tmpfs none /tmp
 fi
 
+# Switch to use iptables instead of nftables (to match the host machine)
+update-alternatives --set iptables  /usr/sbin/iptables-legacy  || true
+update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy || true
+update-alternatives --set arptables /usr/sbin/arptables-legacy || true
+update-alternatives --set ebtables  /usr/sbin/ebtables-legacy  || true
```

Comment:
Shouldn't this happen in the `Dockerfile` instead so we don't accidentally apply this change somewhere less harmless than a container like on a contributor's host machine?

Reply:
It's really a quick hack to see if it did the job
Surprisingly; when I did it as a `RUN` step, it gave a "permission denied; need to be `root`"

Reply:
Oh that's really odd -- AFAICT, we're not using `USER` or anything so it already should be `root`. :sweat:

(And doing `update-alternatives` shouldn't be running `iptables` or anything that might need additional privileges.)

Reply:
That's what I would expect as well; didn't look closely yet, but I suspect it's calling iptables (perhaps just to check if it's installed?)